backend/api/views/main.py

```diff
@@ -74,6 +52,26 @@ def create_person():
     )
 
 
+# function that is called when you visit /census_response
```

Comment:
outdated comment, and also what form is the state going to be passed into this, as a 2-digit ID? should also be commented